### PR TITLE
Get current time with milliseconds precision.

### PR DESCRIPTION
**Pull request by Shylock-Hg**
Close https://github.com/vesoft-inc/nebula-graph/issues/830

### CODE REVIEW

File: src/common/time/TimeUtils.h [hunk 1]
```diff
@@ -278,6 +271,17 @@ class TimeUtils {
 
     static Timezone globalTimezone;
 
+    struct UnixTime {
+        int64_t seconds{0};
+        int64_t milliseconds{0};
```
Comment (laura-ding) [inline src/common/time/TimeUtils.h:276]: here is microseconds

File: src/common/time/TimeUtils.h [hunk 2]
```diff
@@ -278,6 +271,17 @@ class TimeUtils {
...
+    };
+
+    // <seconds, milliseconds>
+    static UnixTime unixTime() {
+        auto ms = WallClock::fastNowInMilliSec();
```
Comment (laura-ding) [inline src/common/time/TimeUtils.h:281]: you can use `fastNowInMicroSec` directly

**Review by laura-ding — COMMENTED** [review]
Could you add some tests?

File: src/common/time/TimeUtils.h (hunk 1 above)
Comment (Shylock-Hg) [reply src/common/time/TimeUtils.h:276]: Milliseconds in fact.

File: src/common/time/TimeUtils.h (hunk 2 above)
Comment (Shylock-Hg) [reply src/common/time/TimeUtils.h:281]: The cypher only provide the milliseconds precision

**Comment by Shylock-Hg** [comment]
> Could you add some tests?

Can't know the current time number, so no better to compare it.

File: src/common/time/TimeUtils.h [hunk 3]
```diff
@@ -112,21 +113,17 @@ class TimeUtils {
...
-        DateTime dt;
-        time_t unixTime = std::time(NULL);
-        if (unixTime == -1) {
-            return Status::Error("Get unix time failed: %s.", std::strerror(errno));
-        }
-        return unixSecondsToDateTime(unixTime - getGlobalTimezone().utcOffsetSecs());
+        auto time = unixTime();
+        auto dt = unixSecondsToDateTime(time.seconds - getGlobalTimezone().utcOffsetSecs());
+        dt.microsec = time.milliseconds * 1000;
+        return dt;
```
Comment (bright-starry-sky) [inline src/common/time/TimeUtils.h:119]: If no errors are returned, change the structure of the return value as better.

Comment (Shylock-Hg) [reply src/common/time/TimeUtils.h:119]: Fixed.

**Review by bright-starry-sky — DISMISSED** [review]
LGTM

**Review by Aiee — APPROVED** [review]

**Review by Aiee — APPROVED** [review]

**Review by yixinglu — APPROVED** [review]

**Pull request merged**